dataland-frontend/src/components/resources/frameworkDataSearch/ThreeLayerDataTable.vue

```diff
@@ -0,0 +1,326 @@
+<template>
+  <div v-show="mapOfCategoryKeysToDataObjectArrays.size > 0">
```

Comment:
Column header for percentage of Total Revenue in Production Sites Modal is missing:

![image](https://github.com/d-fine/Dataland/assets/35459207/a5bb4b65-69ed-42cc-bd91-07bab85b9e6b)

Reply:
Done. And we have a feature that allows manipulating data before sending it to the displaying table.